node.gyp

```diff
@@ -344,11 +344,8 @@
             'src/inspector_socket_server.h',
           ],
           'dependencies': [
-            'deps/v8_inspector/src/inspector/inspector.gyp:standalone_inspector',
-            'v8_inspector_compress_protocol_json#host',
           ],
```

Comment:
That's just how gyp works.  Without an explicit target, the generated `out/Makefile` builds everything.